src/components/trial/TrialScreen.tsx

```diff
@@ -0,0 +1,72 @@
+import { map } from '@firebase/util';
+import React, { useState, useEffect } from 'react';
+
+import { ReactComponent as Trial1 } from '../../svg/Trial-1.svg';
+import { ReactComponent as Trial2 } from '../../svg/Trial-2.svg';
+import { ReactComponent as Trial3 } from '../../svg/Trial-3.svg';
+import { ReactComponent as Trial4 } from '../../svg/Trial-4.svg';
+
+import css from './TrialScreen.module.scss';
+
+interface TrialScreenProps {
+  trial: boolean;
+  trialStage: number;
+  progressTrial: () => void;
+  play: () => void;
+}
+
+const TrialScreen = (props: TrialScreenProps) => {
+  const { trial, trialStage, progressTrial, play } = props;
+
+  if (!trial || trialStage == 3) {
+    document.querySelector('section')?.focus();
+    return null;
+  }
+
+  const renderFigure = (trialStage: number) => {
+    switch (trialStage) {
+      case 2:
+        return <Trial2 className={css.Figures} />;
+      case 4:
+        return <Trial3 className={css.Figures} />;
+      case 5:
+        return <Trial4 className={css.Figures} />;
+    }
+  };
+
+  return trialStage == 1 ? (
```

Comment:
Use === to do a more precise comparison. 
'1' == ' is true, but '1' === 1 is false